ansible/roles/ca-cert/tasks/main.yml

```diff
@@ -30,4 +30,29 @@
 
 - name: Update the certificate
   command: update-ca-certificates
+  when: >
+    ansible_distribution == "SLES" and
+    ansible_distribution_version != "15.3"
```

Comment:
Should this be == 15.2? When we rev to something newer than 15.3, this block will trigger again.

Reply:
Well if I did == 15.2, then it wouldn't run on SLES 15 SP1. I was hoping that SLES would fix update-ca-certificates in any newer version.

Reply:
This just feels like we're kicking this down the road and we'll get bitten by it again at some unknown time. Can we at least add some context (comment, etc) to the task so that when we do hit this again we can know what decisions we made now?

Reply:
I did add a comment to the task right after this one, but I could add it to this one too.